README.md

```diff
@@ -1,17 +1,8 @@
 # Deis Monitor v2
-
-[![Build Status](https://travis-ci.org/deis/monitor.svg?branch=master)](https://travis-ci.org/deis/monitor)
-[![Docker Repository on Quay](https://quay.io/repository/deisci/monitor/status "Docker Repository on Quay")](https://quay.io/repository/deisci/monitor)
```

Comment:
no because we dont produce a monitor image. instead we do 3 images (grafana, telegraf, and influx)